Assets/MixedRealityToolkit-SDK/Features/Boundary/Components/Scripts/PlayAreaVisualizer.cs

```diff
@@ -8,27 +12,14 @@ namespace Microsoft.MixedReality.Toolkit.SDK.Boundary
     /// <summary>
     /// Visualizes the inscribed rectangular play area within the configured boundary.
     /// </summary>
-    public class VisualizePlayArea : MonoBehaviour
+    public class PlayAreaVisualizer : MonoBehaviour
     {
-        [Tooltip("The material to use when visualizing the play area rectangle.")]
-        [SerializeField]
-        private Material playAreaMaterial = null;
-
-        /// <summary>
-        /// The material to use when visualizing the play area rectangle.
-        /// </summary>
-        public Material PlayAreaMaterial
-        {
-            get { return playAreaMaterial; }
-            set { playAreaMaterial = value; }
-        }
-
         [Tooltip("Should the play area be visualized?")]
         [SerializeField]
         private bool isPlayAreaVisualized = true;
 
         /// <summary>
-        /// Gets or sets a value indicating whether or not the play area is visualized.
+        /// Gets or sets a value indicating whether or not the play area is / should be visualized.
```

Comment:
`is or will be`?

Reply:
`is (should be)`?

Reply:
it's should be, since the user can specify to hide the plane